SamMou.Api/Controllers/WeatherForecastController.cs

```diff
@@ -27,6 +27,7 @@ public async Task<IActionResult> GetAll()
         {
             try
             {
+                _logger.LogInformation("Called GetAll");
```

Comment:
Do you have example output? It'll show the other metadata such as which method called this, right?

Reply:
yup you can go to application insights to see the output

Reply:
![image](https://user-images.githubusercontent.com/26915914/124215824-633bc100-db27-11eb-85b6-949dd1517d40.png)

Reply:
Saw it! What's the retention policy for our logs?